src/index.css

```diff
@@ -467,3 +467,35 @@ textarea {
 .terminal .fullscreen {
   background-color: #859900;
 }
+
+.github-corner:hover .octo-arm {
+  animation: octocat-wave 560ms ease-in-out;
+}
+
+
```

Comment:
Could you remove the duplicate empty line? If you run `npm test`, css is checked, an you’ll see a warning about this!